mopidy/backend.py

```diff
@@ -29,38 +68,40 @@ class Backend:
     #: The library provider. An instance of
     #: :class:`~mopidy.backend.LibraryProvider`, or :class:`None` if
     #: the backend doesn't provide a library.
-    library = None
+    library: Optional["LibraryProvider"] = None
 
     #: The playback provider. An instance of
     #: :class:`~mopidy.backend.PlaybackProvider`, or :class:`None` if
     #: the backend doesn't provide playback.
-    playback = None
+    playback: Optional["PlaybackProvider"] = None
 
     #: The playlists provider. An instance of
     #: :class:`~mopidy.backend.PlaylistsProvider`, or class:`None` if
     #: the backend doesn't provide playlists.
-    playlists = None
+    playlists: Optional["PlaylistsProvider"] = None
 
     #: List of URI schemes this backend can handle.
-    uri_schemes = []
+    uri_schemes: List[str] = []
```

Comment:
In theory we could have a stricter scheme type, but probably overkill. Also I still think we should switch to URI prefixes at some point.

Reply:
Fixed.